pandas/io/excel/_xlrd.py

```diff
@@ -0,0 +1,127 @@
+from datetime import time
+from distutils.version import LooseVersion
+from io import UnsupportedOperation
+
+import numpy as np
+
+import pandas.compat as compat
+from pandas.compat import range, zip
+
+from pandas.io.common import (
+    _is_url, _urlopen, get_filepath_or_buffer)
+from pandas.io.excel._base import _BaseExcelReader
+
+
+class _XlrdReader(_BaseExcelReader):
+
+    def __init__(self, filepath_or_buffer):
+        """Reader using xlrd engine.
+
+        Parameters
+        ----------
+        filepath_or_buffer : string, path object or Workbook
+            Object to be parsed.
+        """
+        err_msg = "Install xlrd >= 1.0.0 for Excel support"
+
+        try:
+            import xlrd
+        except ImportError:
+            raise ImportError(err_msg)
+        else:
+            if xlrd.__VERSION__ < LooseVersion("1.0.0"):
+                raise ImportError(err_msg +
+                                  ". Current version " + xlrd.__VERSION__)
+
+        from pandas.io.excel._base import ExcelFile
```

Comment:
Here's the other part of the circular reference which I had to move to the class __init__ to resolve